### ISSUE TITLE
UI inconsistency and bug Fix

### ISSUE THREAD

**Issue by nikiljos** [foodfolio/foodfolio.github.io#8]
We have some UI Inconsistency and some bugs in some of the subpages(excluding stores and locations). We need to fix them

### Some bugs noted
- Responsiveness issue and floating footer in `About Us` Page
- Minor UI Fixes and media queries in `Promotions` Page
- Invalid Image URLs and Responsivess issue in `Blogs` Page

**Comment by nikiljos** [comment]
Pls drop a comment if u can take up any of these fixes.

**Comment by nikiljos** [comment]
@TukadiyaRaju Pls take a look into it. Also pls note down other UI bugs that we r having

**Comment by rajuTK9** [comment]
Yeah, @nikiljos, there are a few more things needed to be fixed with UI:
1. Nav bar inconsistency on different pages.
2. Footer inconsistency on different pages.
3. Inappropriate hover effect on '<a></a>' links.
4. Link to the home page is not applied to the logo inside the navbar on many pages.
5. Inconsistent image layout on the restaurants having more than one menu image.
6. Favicon is only applied to the home page.

That's all that I could find. I think people can take these bugs one by one and fix them working individually.
